openedx/core/djangoapps/content_libraries/tasks.py

```diff
@@ -300,7 +300,13 @@ def duplicate_children(
     # Then, copy over any overridden settings the course author may have applied to the blocks.
     source_block = store.get_item(BlockUsageLocator.from_string(source_block_id))
     with store.bulk_operations(source_block.scope_ids.usage_id.context_key):
-        _copy_overrides(store=store, user_id=user_id, source_block=source_block, dest_block=dest_block)
+        try:
+            TASK_LOGGER.info('Copying Overrides for %s', dest_block.scope_ids.usage_id)
+            _copy_overrides(store=store, user_id=user_id, source_block=source_block, dest_block=dest_block)
+        except Exception as exception:  # pylint: disable=broad-except
+            TASK_LOGGER.exception('Error importing children for %s', dest_block.scope_ids.usage_id, exc_info=True)
+            if self.status.state != UserTaskStatus.FAILED:
+                self.status.fail({'raw_error_msg': str(exception)})
```

Comment:
* I had similar logging going locally when I was working on the original PR, and I found it helpful to include the source id here too.
* 'importing children' -> 'copying overrides'
* using the function args (which are already strs) will make the output nicer. e.g., `block-v1:a+b+c+d@...` instead of `BlockUsageLocator(org='a', course='b', ...)'`.

```suggestion
        try:
            TASK_LOGGER.info('Copying Overrides from %s to %s', source_block_id, dest_block_id)
            _copy_overrides(store=store, user_id=user_id, source_block=source_block, dest_block=dest_block)
        except Exception as exception:  # pylint: disable=broad-except
            TASK_LOGGER.exception('Error Copying Overrides from %s to %s', source_block_id, dest_block_id)
            if self.status.state != UserTaskStatus.FAILED:
                self.status.fail({'raw_error_msg': str(exception)})
```